src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -10,25 +10,31 @@ Below is an example of how you would have previously in v5 interacted with the A
 import { CognitoUser } from '@aws-amplify/auth';
 import { Auth } from 'aws-amplify';
 
-const handleSignIn = async ({username, password}: {
-    username: string,
-    password: string
+const handleSignIn = async ({
+  username,
+  password
+}: {
+  username: string;
+  password: string;
 }) => {
-    const user: CognitoUser = Auth.signIn(username, password);
-}
+  const user: CognitoUser = Auth.signIn(username, password);
+};
 ```
 
 The following is an example of how you would accomplish the same functionality with v6, using our new imports and API surface.
 
 ```ts
-import { signIn, SignInInput, SignInOutput } from 'aws-amplify/auth';
-
-const handleSignIn = async({username, password}: {
-    username: string,
-    password: string
+import { signIn } from 'aws-amplify/auth';
+
+const handleSignIn = async ({
+  username,
+  password
+}: {
+  username: string;
```

Comment:
suggestion not-blocking: wondering if we can import the types like `AuthSignInInput` in ts snippets if that makes sense